project_euler/problem_119/sol1.py

```diff
@@ -0,0 +1,44 @@
+"""
+"https://projecteuler.net/problem=119"
+
+Name: Digit power sum
+
+The number 512 is interesting because it is equal to the sum of its digits
+raised to some power: 5 + 1 + 2 = 8, and 83 = 512. Another example of a number
+with this property is 614656 = 284. We shall define an to be the nth term of
+this sequence and insist that a number must contain at least two digits to have a sum.
+You are given that a2 = 512 and a10 = 614656. Find a30
+"""
+
+
+import math
+
+
+def digit_sum(n: int) -> int:
+    """
+    Returns the sum of the digits of the number
+    """
+    sum = 0
+    while n > 0:
+        sum += n % 10
+        n //= 10
+    return sum
```

Comment:
```suggestion
    return sum([int(digit) for digit in str(n)])
```

Reply:
replaced with the suggested simplified version.